pytools/tag.py

```diff
@@ -362,6 +362,18 @@ class IgnoredForEqualityTag(Tag):
 # }}}
 
 
+# {{{ IgnoredForPropagationTag
+
+class IgnoredForPropagationTag(UniqueTag):
+    """
+    Used to signify that a Taggable should *not* have any tags propagated along
```

Comment:
Add some context (in terms of referencing the functions added in this PR) on what "propagation" is.